gulpfile.js

```diff
@@ -39,8 +41,7 @@ var port         = 8080;
     ===================================================== */
 
 gulp.task('clean', function() {
-  return gulp.src(demo, {read: false})
-  .pipe(clean());
+  return del([demo, dist])
```

Comment:
is del() === clean() here ? What is the the benifit of use del() instead of clean()?

Reply:
del is quicker and easy to delete multiple files/folders